dashboard/views.py

```diff
@@ -66,9 +66,63 @@ def get(self, request):
 
 class WarningsInstance_View(View):
 
-    def get(self, request):
+    def get(self, request, **kwargs):
+
+        cap_id = kwargs['cap_id']
+        cap_instance = CAPInstance.objects.get(cap_id=cap_id)
+
+        # Grab Masterlist Atrributes
+        brokenlist = self.get_broken_urls(cap_instance)
+        retaglist = self.get_retag(cap_instance)
+        nimlist = self.get_nim(cap_instance)
```

Comment:
These three functions namely  `get_broken_urls(cap_instance)`, `get_retag(cap_instance)`, `get_nim(cap_instance)` look identical. Can we instead make a function like: 
```
def get_metrics(cap_id, Model, columns_to_fetch):
    return Model.objects.filter(cap_id=cap_id).values(*columns_to_fetch)
    
```

Reply:
Have consolidated two of them - but have questions about the last. Will follow up